.github/workflows/approved_label.yml

```diff
@@ -0,0 +1,14 @@
+on: pull_request_review
```

Comment:
If we're going to use the [Probot Stale](https://github.com/apps/stale) app, then this will be good, as the app won't mark as stale the PRs that have the `approved` label. 🙂

Reply:
First of all: I :heart: labels! But I'm not a fan of duplicated functionality. GitHub already provides **many** options to filter by review status:
![image](https://user-images.githubusercontent.com/8781699/109779932-c4e63000-7c06-11eb-94fc-51485a5963bb.png)

I don't see an advantage of having a `approved` label indicating the same what GitHub already provides. Or am I missing something?

Auto adding the `feature:icon` label is a nice enhancement! Maybe someone can take a look at [pull request templates](https://docs.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository)? They might allow to achieve the same result (without a extra workflow). With issue templates (which we [already use](https://github.com/devicons/devicon/edit/master/.github/ISSUE_TEMPLATE/icon-request.md)) it's possible to define a label which is added automatically.

Reply:
> I don't see an advantage of having an `approved` label indicating the same what GitHub already provides

Well, to me it is easier to use the `approved` label. Sometimes, we merge PRs, but we don't approve them; still, we can add the `approved` label, and it'll be easier to track them (without being approved).

It's your decision if you want to have it in Devicon or not, but to me, it's a good idea. 🙂